README.md

```diff
@@ -1 +1,9 @@
-# Hello-World
\ No newline at end of file
+# Hello-World
+
+> This project is to interact with branches and learn how to pull request 
+
+## Built With
+
+- Html
+- CSS
+- Linters
```

Comment:
- [ ] you should also add the other sections from the template readme file like your social media accounts, how can one install your projects locally and customize the other sections to your own please try to do that